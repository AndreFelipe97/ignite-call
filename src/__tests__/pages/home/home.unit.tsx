import Home from "@/pages/home";
import { render, screen } from "@testing-library/react";

describe("<Home />", () => {
  it("should render correctly home page", () => {
    render(<Home />);

    const homePageTitle = screen.getByText("Agendamento descomplicado");
    const homePageSubtitle = screen.getByText(
      "Conecte seu calend√°rio e permita que as pessoas marquem agendamentos no seu tempo livre"
    );

    expect(homePageTitle).toBeInTheDocument();
    expect(homePageSubtitle).toBeInTheDocument();
    expect(homePageTitle).toMatchSnapshot();
    expect(homePageSubtitle).toMatchSnapshot();
  });
});
