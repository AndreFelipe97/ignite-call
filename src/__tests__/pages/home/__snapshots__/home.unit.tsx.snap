// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly home page 1`] = `
<h1
  class="c-btBDqU c-btBDqU-idUsPu-size-4xl"
>
  Agendamento descomplicado
</h1>
`;

exports[`<Home /> should render correctly home page 2`] = `
<p
  class="c-kQRPbk c-kQRPbk-iPGoSV-size-xl"
>
  Conecte seu calend√°rio e permita que as pessoas marquem agendamentos no seu tempo livre
</p>
`;
